ludwig/features/image_feature.py

```diff
@@ -54,33 +56,25 @@
     read_image_from_bytes_obj,
     read_image_from_path,
     resize_image,
+    torchvision_model_registry,
 )
 from ludwig.utils.misc_utils import set_default_value
 from ludwig.utils.types import Series, TorchscriptPreprocessingInput
 
 logger = logging.getLogger(__name__)
 
 
-# TODO(shreya): Confirm if it's ok to do per channel normalization
-# TODO(shreya): Also confirm if this is being used anywhere
-# TODO(shreya): Confirm if ok to use imagenet means and std devs
-image_scaling_registry = {
-    "pixel_normalization": lambda x: x * 1.0 / 255,
-    "pixel_standardization": partial(
-        torchvision.transforms.functional.normalize, mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]
-    ),
-}
-
-
 class _ImagePreprocessing(torch.nn.Module):
     """Torchscript-enabled version of preprocessing done by ImageFeatureMixin.add_feature_data."""
 
-    def __init__(self, metadata: Dict[str, Any]):
+    def __init__(self, metadata: Dict[str, Any], tv_transforms: Optional[torch.nn.Module] = None):
         super().__init__()
         self.height = metadata["preprocessing"]["height"]
         self.width = metadata["preprocessing"]["width"]
         self.num_channels = metadata["preprocessing"]["num_channels"]
         self.resize_method = metadata["preprocessing"]["resize_method"]
+        self.torchvision_model_id = metadata["preprocessing"].get("torchvision_model_id")
+        self.tv_transforms = tv_transforms
```

Comment:
would it be possible to just check if `tv_transforms is None`? If so, do we need to store `torchvision_model_id` here?

Reply:
made recommended changes